packages/web-components/src/components/ic-navigation-button/ic-navigation-button.tsx

```diff
@@ -162,7 +162,11 @@ export class NavigationButton {
           {...this.inheritedAttributes}
         >
           {label}
-          <slot slot="icon" name="icon"></slot>
+          {variant === "icon" ? (
+            <slot name="icon"></slot>
```

Comment:
i think the `slot="icon"` is needed to tell it which slot in ic-button to use. Should it be `slot="left-icon"` now?

Reply:
Yeah I think that fixes it. Will update now 👍